pkgs/applications/misc/keyleds/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, libuv
+, libX11
+, libXi
+, libyaml
+, luajit
+, udev
+}:
+
+stdenv.mkDerivation rec {
+  pname = "keyleds";
+  version = "2022-07-03";
+
+  src = fetchFromGitHub {
+    owner = "NicBOMB";
+    repo = pname;
+    rev = "5445fe3137813eda6d727cf5874ec9e40873129f";
+    sha256 = "sha256-yEAQONQwl685JQqCb8Ba8p3LAn0ogCSjPXrEdC7MEAQ=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    libuv
+    libX11
+    libXi
+    libyaml
+    luajit
+    udev
+  ];
```

Comment:
Looks like something should stay in `buildInputs`. Would you please try setting `strictDeps = true`?

Reply:
Done :)

Reply:
That's fine for me.